addons/suggest_locations/tests/common.py

```diff
@@ -0,0 +1,446 @@
+# -*- coding: utf-8 -*-
+from odoo.tests import common
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+@common.at_install(False)
+@common.post_install(True)
+class BaseUDES(common.SavepointCase):
```

Comment:
This is not probably needed, we have dependency from udes_stock which sets up BaseUDES. We should only add extra stuff on that class here instead of redefining.